cmd/cosign/cli/options/verify.go

```diff
@@ -34,6 +35,9 @@ func (o *CommonVerifyOptions) AddFlags(cmd *cobra.Command) {
 
 	cmd.Flags().StringVar(&o.TSACertChainPath, "timestamp-cert-chain", "",
 		"path to certificate chain PEM file for the Timestamp Authority")
+
+	cmd.Flags().BoolVar(&o.SkipTlogVerify, "skip-tlog-verify", false,
```

Comment:
I’d prefer a scarier name, I really want to avoid people setting this accidentally. Can we preface it with “insecure”?

Reply:
I added in both the flag and the description.